django_server/system_settings/FEATURES.md

```
@@ -0,0 +1,199 @@
+- chat_sent
+- websocket_chat_sent
+- chat_feedback
+- chat_exported
+
+### System Events
+- system_setting_changed
+- api_key_generated
+- invite_generated
+- invite_claimed
+
+## System Status Information
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)